stable/jenkins/values.yaml

```diff
@@ -173,13 +164,62 @@ Master:
   # become the content of the configuration yaml file.  The first line after this is a JCasC root element, eg jenkins, credentials,
   # etc.  Best reference is https://<jenkins_url>/configuration-as-code/reference.  The example below creates a welcome message:
   JCasC:
-    enabled: false
-    PluginVersion: 1.5
-    SupportPluginVersion: 1.5
+    enabled: true
+    PluginVersion: 1.6
+    SupportPluginVersion: 1.6
     ConfigScripts:
       welcome-message: |
         jenkins:
           systemMessage: Welcome to our CI\CD server.  This Jenkins is configured and managed 'as code'.
+      executors-on-master: |
+        jenkins:
+          numExecutors: 0
+      agent-protocols: |
+        jenkins:
+          agentProtocols:
+            - "JNLP4-connect"
+      csrf: |
+        jenkins:
+          crumbIssuer:
+            standard:
+               excludeClientIPFromCrumb: true
+      remoting-cli: |
+        security:
+            remotingCLI:
+                  enabled: false
+      kubernetes-slave: |
+        jenkins:
+          clouds:
+            - kubernetes:
+                name: "jenkins-slave"
+                serverUrl: https://kubernetes.default
+                skipTlsVerify: true
```

Comment:
This should be `skipTlsVerify: false` as it was before: https://github.com/helm/charts/blob/master/stable/jenkins/templates/config.yaml#L131

Reply:
Addressed in [commit](https://github.com/helm/charts/pull/11706/commits/e0faa81d38915f54a1c39c760563ba7ce7e04503)